data/db.js

```diff
@@ -10,7 +10,6 @@ const connection = mysql.createConnection({
 connection.connect();
 
```

Comment:
After the connection is created, this code needs a 'CREATE DATABASE IF NOT EXISTS agent' statement.